gnome-power-statistics.sgml:1: parser error : StartTag: invalid element name

Index: man/meson.build
--- man/meson.build.orig
+++ man/meson.build
@@ -1,4 +1,4 @@
-docbook2man = find_program('docbook2man', required : false)
+docbook2man = find_program('docbook2man-disable', required : false)
 if docbook2man.found()
   custom_target('gnome-power-statistics-man',
     output : 'gnome-power-statistics.1',
